cortex/cli.py

```
@@ -261,6 +267,306 @@ def rollback(self, install_id: str, dry_run: bool = False):
+                        if not all_passed:
+                            print("\n[WARNING] Some verification checks failed:")
+                            for cmd, passed in verify_results.items():
+                                status = "[OK]" if passed else "[FAIL]"
+                                print(f"  {status} {cmd}")
+                    
+                    # Run post-install commands once
+                    if template.post_install:
+                        self._print_status("[*]", "Running post-installation steps...")
+                        print("\n[*] Post-installation information:")
+                        for cmd in template.post_install:
+                            subprocess.run(cmd, shell=True)
```

Ruff (0.14.5)

`subprocess` call with `shell=True` identified, security issue

(S602)